hadoop-hdds/interface-admin/src/main/proto/ScmAdminProtocol.proto

```diff
@@ -302,7 +302,11 @@ message SCMCloseContainerRequestProto {
 }
 
 message SCMCloseContainerResponseProto {
-  // Empty response
+  enum Error {
+    OK = 1;
+    CONTAINER_ALREADY_CLOSED = 2;
+  }
+  required Error errorCode = 1;
```

Comment:
I think we can try to be backwards compatible.

```suggestion
  optional Error errorCode = 1;
```

Also, since `OK` is also part of the enum, `Status` might be a better name.